app/js/reducers/addDrugOrderReducer.js

```diff
@@ -0,0 +1,27 @@
+import {
+  POST_DRUG_ORDER_SUCCESS,
+  POST_DRUG_ORDER_FAILURE,
+} from '../../../app/js/actions/actionTypes';
+
+const initialState = {
+  addedOrder: {},
+  error: null,
+};
+
+const addDrugOrderReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case POST_DRUG_ORDER_SUCCESS:
+      return {
+        ...state,
+        addedOrder: action.response,
```

Comment:
So for example, this just changes the state to hold one object at a time...in other words, what happens when you post multiple orders?

Reply:
so it becomes an object or array of order objects?

Reply:
@zeze111 An object having other objects plus an array of orders. 

<img width="887" alt="screen shot 2018-05-07 at 2 33 34 pm" src="https://user-images.githubusercontent.com/19909781/39699859-af46e68e-5203-11e8-90f4-116370d6a91a.png">